src/vmm/src/lib.rs

```diff
@@ -100,6 +100,11 @@ pub const FC_EXIT_CODE_BAD_CONFIGURATION: ExitCode = 152;
 /// Command line arguments parsing error.
 pub const FC_EXIT_CODE_ARG_PARSING: ExitCode = 153;
 
+/// Timeout in milliseconds used in recv_timeout, when waiting for a vcpu response on
+/// Pause/Resume/Save/Restore. A high enough limit that should not be reached during normal usage
+/// used to detect a potential vcpu deadlock.
+pub const RECV_TIMEOUT_MS: u64 = 30000;
```

Comment:
nitish:

you could also define the `Duration` here instead of building it each time:
```suggestion
pub const RECV_TIMEOUT_MS: u64 = 30000;
pub const RECV_TIMEOUT = Duration::from_millis(RECV_TIMEOUT_MS);
```

Reply:
good idea, done 👍🏻